frontend/src/lib/utils.tsx

```diff
@@ -38,16 +38,16 @@ export function toParams(obj: Record<string, any>): string {
 }
 
 export function fromParams(): Record<string, any> {
-    return window.location.search === ''
+    return !window.location.search
         ? {}
         : window.location.search
               .slice(1)
               .split('&')
-              .reduce((a, b) => {
-                  b = b.split('=')
-                  a[b[0]] = decodeURIComponent(b[1])
-                  return a
-              }, {})
+              .reduce((paramsObject, paramString) => {
```

Comment:
**Non-blocking.** Maybe worth to using the default logic on kea-router?

Reply:
In another PR.